src/Header.js

```diff
@@ -0,0 +1,10 @@
+const Header = () => {
+  return (
+    <>
+      <h1 className="heading__title--main">GOT ANY PLANS?</h1>
+      <h2 className="heading__title--sub">Plan your week!</h2>
+    </>
```

Comment:
As I mentioned above. The Fragments in this component could be the `<header>` tags that are currently in App.js.